### PR TITLE
LOCO with constraints and QS fitting

### PR DESCRIPTION
**Pull request by murilobalves**
- Including constraints in the total delta K as suggested in the article "LOCO with constraints and improved fitting technique" by Xiaobiao Huang, James Safranek, 2009 
- Skew quadrupoles added as possible knobs to the fitting

### CODE REVIEW

File: apsuite/commissioning_scripts/loco.py [hunk 1]
```diff
@@ -1241,6 +1311,23 @@ def _handle_dip_fit_ks(self, fname_jloco_ks_dip):
                 self._jloco_ks_dip = self._convert_dict2array(
                     jloco_ks_dip_dict, 'dipole', is_normal=False)
 
+    def _handle_skewquad_fit_ks(self, fname_jloco_ks_skewquad):
+        # calculate Ks jacobian for skew quadrupoles
+        if self.config.fit_skew_quadrupoles:
+            if fname_jloco_ks_skewquad is None:
+                print('calculating skew quadrupoles ksmatrix...')
+                t0 = time.time()
```
Comment (xresende) [inline apsuite/commissioning_scripts/loco.py:1319]: pylint 3-letter minimum name rule. suggestions: `time0`, `dtime`

Comment (murilobalves) [reply apsuite/commissioning_scripts/loco.py]: ok, I followed all pylint suggestions in the last commit, except one: `pylint(too-many-lines)`. But I hope to solve that problem soon, breaking the loco.py in smaller subfiles.

File: apsuite/commissioning_scripts/loco.py [hunk 2]
```diff
@@ -3,10 +3,10 @@
 from copy import deepcopy as _dcopy
 import time
 import pickle as _pickle
+from siriuspy.namesys import SiriusPVName as _PVName
```
Comment (fernandohds564) [inline apsuite/commissioning_scripts/loco.py:6]: `siriuspy` must come after `numpy`...

Comment (murilobalves) [reply apsuite/commissioning_scripts/loco.py]: ok

File: apsuite/commissioning_scripts/loco.py (hunk 1 above)
Comment (xresende) [reply apsuite/commissioning_scripts/loco.py]: Yes, `too-many-lines`, `too-many-arguments` and `too complex` are harder to follow. we comply gradually. (in general I agree with these too! every time I took the time to fix `too complex` I ended up with the filling that code was easier to read)

Comment (xresende) [reply apsuite/commissioning_scripts/loco.py]: I'll keep this unresolved so you have a better change to read my comment. but you can resolve it and merge after you read this.

**Pull request merged**